bundles/org.openhab.binding.pushover/src/main/java/org/openhab/binding/pushover/internal/handler/PushoverAccountHandler.java

```diff
@@ -143,10 +148,10 @@ public boolean sendMessage(PushoverMessageBuilder messageBuilder) {
         if (connection != null) {
             try {
                 return connection.sendMessage(messageBuilder);
-            } catch (PushoverCommunicationException e) {
+            } catch (CommunicationException e) {
                 // do nothing, causing exception is already logged
-            } catch (PushoverConfigurationException e) {
-                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, e.getMessage());
+            } catch (ConfigurationException e) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, e.getLocalizedMessage());
```

Comment:
Same here, please use `e.getRawMessage()`

Reply:
Hmm, this text is user-oriented (exposed in the UI), so isn't the idea exactly to use the localized text?

Reply:
Yes, it is and to get the localized text in updateStatus, while `@text` was used as message key in exceptions, you have to call getRawMessage(). Calling getLocalizedMessage() without any params will not return the localized text. 
See that PR: https://github.com/openhab/openhab-core/pull/2549 for the full discussion. 
Ideally, I should create a chapter to document how to use properly AbstractI18nException.

Reply:
> See that PR: [openhab/openhab-core#2549](https://github.com/openhab/openhab-core/pull/2549) for the full discussion.

Thanks for this explanation, very useful.

Reply:
Thanks. I kept it wrong in my mind. I probably have to fix it for OpenWeaterhMap too after https://github.com/openhab/openhab-addons/pull/11949.

Reply:
Yes, I can confirm you have to fix OWN binding.